tasks/Ubuntu.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: add repository from PPA and install its signing key
+  apt_repository:
+    repo: "{{ haproxy_ppa }}"
```

Comment:
There is no PPA for Debian. Also it should be just called repository. Take a look on: https://haproxy.debian.net/#?distribution=Debian&release=stretch&version=1.8